bin/collator/Cargo.toml

```diff
@@ -131,7 +132,7 @@ sc-cli = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v
 sc-service = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.33", optional = true }
 
 [features]
-default = ["sc-cli", "polkadot-cli", "sc-service", "sc-service/rocksdb"]
+default = ["sc-cli", "polkadot-cli", "sc-service", "sc-service/rocksdb", "runtime-benchmarks"]
```

Comment:
For convenience, temporarily add runtime-benchmarks as default.

Reply:
Please remove this part now :slightly_smiling_face: